docs/lib/project-setup/fragments/android/create-application/20_gradle.md

```diff
@@ -28,7 +36,7 @@ allprojects {
 apply plugin: 'com.amplifyframework.amplifytools'
 ```
 
-- Add the line `classpath 'com.amplifyframework:amplify-tools-gradle-plugin:1.0.0'` within the `dependencies` block which is within the `buildscript` block
+- Add the line `classpath 'com.amplifyframework:amplify-tools-gradle-plugin:1.0.1'` within the `dependencies` blocks in both `buildscript` and `allprojects` blocks
 - Add the line `mavenCentral()` within the `repositories` block which is within the `allprojects` block
```

Comment:
This doesn't match what we have, above -- I think it should read:
```suggestion
- Add the line `classpath 'com.amplifyframework:amplify-tools-gradle-plugin:1.0.1'` to the `dependencies` block inside `buildscript`.
- Add `mavenCentral()` to each of the `repositories` blocks under `buildscript` and `allprojects`.
```

Reply:
Ah yeah, I blew it. Will fix.